orienteer-users/src/main/java/org/orienteer/users/web/DefaultRegistrationPage.java

```diff
@@ -54,17 +52,6 @@ public DefaultRegistrationPage() {
      */
     public DefaultRegistrationPage(PageParameters parameters) {
         super(parameters);
-        String id = parameters.get(RegistrationResource.PARAMETER_ID).toOptionalString();
```

Comment:
Is this deletion breaking other usages of orienteer-users?

Reply:
We update user status in registration resource not in page. First implementation for update user status in registration page was wrong, because page is just view.